01-data-structures/01-introduction-to-data-structures/screen/screen.rb

```diff
@@ -6,18 +6,23 @@ class Screen
   attr_accessor :matrix
 
   def initialize(width, height)
+    @width = width
+    @height = height
+    @matrix = Array.new(@height){Array.new(@width)}
   end
 
   # Insert a Pixel at x, y
   def insert(pixel, x, y)
+    @matrix[x][y] = pixel
   end
 
   def at(x, y)
+    @matrix[x][y]
   end
 
   private
 
   def inbounds(x, y)
```

Comment:
We talked about this working in spite of not implementing this due to the way arrays work in Ruby (returns `nil` for outside boundaries). What if that weren't the case? How would you implement `inbounds()`?